### DISCUSSION TITLE
G7 QR code scan not possible because of broken camera

### DISCUSSION THREAD

**Discussion by wadert3** [NightscoutFoundation/xDrip#2970] [Q&A]
I can't scan the qr code on the docs because my camera is broken, is there perhaps and import file or a settings page that I can manually fill out within the app to work around this? Just attached my first one and need to get it figured, let me know, thanks!

**Comment by Navid200** [comment]
You need to scan it with auto config.
Or, you can use Dexcom app.

**Comment by Navid200** [comment]
I have not tested this and it may fail:

Use an alternate phone that has a working camera.  Install xDrip on it.  Scan the QR code.  Save the xDrip settings file.  Transfer the file to your phone.  Load the settings file into xDrip.

I have no idea if this will work.  It may not.  But, what have you got to lose trying it?

**Reply by actrapid** [reply]
Thanks for the Information!
I did this procedure for my Android13 Watch (not WarOS), and it worked fine for me.

**Comment by Navid200** [comment]
Starting with the August 2, 2024 release, there is an option to scan a QR code from an image file saved in storage.
